buidler.config.js

```diff
@@ -0,0 +1,28 @@
+const { task, usePlugin } = require('@nomiclabs/buidler/config')
+const createDisputedVote = require('./lib/create-disputed-vote')
```

Comment:
Oh nice, I thought you'd leave these as scripts but adding them as a task is nice too!

Reply:
yea :D